package.json

```diff
@@ -77,10 +77,10 @@
     ]
   },
   "dependencies": {
-    "@aws-sdk/client-s3": "^3.621.0",
-    "@ministryofjustice/frontend": "^2.1.3",
```

Comment:
anything worth noting from 2.1.3 - > 2.2.0 here?

Reply:
They just added the datepicker component https://github.com/ministryofjustice/moj-frontend/blob/main/CHANGELOG.md 👍